queue_job/tests/common.py

```diff
@@ -1,12 +1,12 @@
 # Copyright 2019 Camptocamp
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 import doctest
+import logging
+import sys
 from contextlib import contextmanager
 
 import mock
 
-from odoo.tests import BaseCase, tagged
-
 # pylint: disable=odoo-addons-relative-import
 from odoo.addons.queue_job.job import Job
```

Comment:
If we don't have the conflict here, maybe we should rebase 14.0-nova?

Reply:
I agree, I shall rebase 14.0-nova